src/pages/[platform]/build-a-backend/data/custom-business-logic/search-and-aggregate-queries/index.mdx

```diff
@@ -843,7 +843,7 @@ dynamodb-pipeline:
 // highlight-start
 // Create a CloudWatch log group
 const logGroup = new logs.LogGroup(dataStack, "LogGroup", {
-  logGroupName: "/aws/vended-logs/OpenSearchService/pipelines/1",
+  logGroupName: "/aws/vendedlogs/OpenSearchService/pipelines/1",
```

Comment:
`vendedlogs` is  not passing the spellcheck, this needs to be added to https://github.com/aws-amplify/docs/blob/main/cspell.json to pass the spellcheck

Reply:
added to spellcheck.